ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/LoginController.java

```diff
@@ -56,6 +56,13 @@ public void login(Context ctx) {
     }
   }
 
+  /** handle logout post request. */
+  public void logout(Context ctx) {
+    ctx.consumeSessionAttribute("user");
```

Comment:
Why need this？

Reply:
Do you mean it(ctx.consumeSessionAttribute("user")) is redundant?